test/functional/shell/bang_filter_spec.lua

```diff
@@ -0,0 +1,69 @@
+-- Specs for bang/filter commands
+
+local helpers = require('test.functional.helpers')
+local feed, execute, clear = helpers.feed, helpers.execute, helpers.clear
+local wait = helpers.wait
+
+local Screen = require('test.functional.ui.screen')
+
+
+describe('issues', function()
+  local screen
+
+  before_each(function()
+    clear()
+    os.execute('rm -rf bang_filter_spec')
```

Comment:
`helpers.rmdir()` is a nice robust alternative that will work on Windows as well.